src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,7 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        String x = Integer.toBinaryString(value);
```

Comment:
you should write your own implementation

Reply:
done